RIOT-OS/RIOT
# examples/lorawan: add LoRaWAN keys to DOCKER_ENV_VARS

### Contribution description

For this application, KEYS are meant to be set in the environment or overridden in the Makefile. If the first approach is used they will not be passed to DOCKER, this PR adds the variables to `DOCKER_ENV_VARS` only for this application. This fixes the failing test in https://github.com/fjmolinas/RIOT/runs/3937818214?check_suite_focus=true which succeeds locally (when not building in docker)

### Testing procedure

Just check that if set the environment variables are passed to DOCKER.

with this pr:

```
11-lorawan/test_spec11.py::test_task01[nodes0] Launching build container using image "riot/riotbuild:latest".
docker run --rm --tty --user $(id -u) -v '/usr/share/zoneinfo/Europe/Paris:/etc/localtime:ro' -v '/home/francisco/workspace/RIOT:/data/riotbuild/riotbase:delegated' -e 'RIOTBASE=/data/riotbuild/riotbase' -e 'CCACHE_BASEDIR=/data/riotbuild/riotbase' -e 'BUILD_DIR=/data/riotbuild/riotbase/build' -e 'RIOTPROJECT=/data/riotbuild/riotbase' -e 'RIOTCPU=/data/riotbuild/riotbase/cpu' -e 'RIOTBOARD=/data/riotbuild/riotbase/boards' -e 'RIOTMAKE=/data/riotbuild/riotbase/makefiles'      -e 'DEVEUI=70B3D57ED00463E7' -e 'APPEUI=0000000000000000' -e 'APPKEY=782D212CE97434EAB523EA7F98A70F52' -e 'BOARD=b-l072z-lrwan1'  -w '/data/riotbuild/riotbase/examples/lorawan/' 'riot/riotbuild:latest' make      
Building application "lorawan" for "b-l072z-lrwan1" with MCU "stm32".
```

without:

```
11-lorawan/test_spec11.py::test_task01[nodes0] Launching build container using image "riot/riotbuild:latest".
docker run --rm --tty --user $(id -u) -v '/usr/share/zoneinfo/Europe/Paris:/etc/localtime:ro' -v '/home/francisco/workspace/RIOT:/data/riotbuild/riotbase:delegated' -e 'RIOTBASE=/data/riotbuild/riotbase' -e 'CCACHE_BASEDIR=/data/riotbuild/riotbase' -e 'BUILD_DIR=/data/riotbuild/riotbase/build' -e 'RIOTPROJECT=/data/riotbuild/riotbase' -e 'RIOTCPU=/data/riotbuild/riotbase/cpu' -e 'RIOTBOARD=/data/riotbuild/riotbase/boards' -e 'RIOTMAKE=/data/riotbuild/riotbase/makefiles'      -e 'BOARD=b-l072z-lrwan1'  -w '/data/riotbuild/riotbase/examples/lorawan/' 'riot/riotbuild:latest' make      
Building application "lorawan" for "b-l072z-lrwan1" with MCU "stm32".
```

### Issues/PRs references

Spotted in #17009
